src/classes/OPP_OpportunityContactRoles_TDTM.cls

```diff
@@ -654,4 +677,34 @@ public without sharing class OPP_OpportunityContactRoles_TDTM extends TDTM_Runna
     }
 
 
+    /*********************************************************************************************************
+    * @description Populates a Map of Account Id to Opportunity Ids.
+    * @param currentOpportunities The Opportunities in the 'new' trigger context variable.
+    * @param oppAccountDetails A Map containing the Opportunities' related Accounts.
+    * @return Map<Id, Id> contains associated Primary Contacts and Opportunity Ids.
+    **********************************************************************************************************/
+    @testVisible
+    private static Map<Id, List<Id>> populateAccountToOpportunities(List<Opportunity> currentOpportunities, Map<Id, Account> oppAccountDetails) {
+        Map<Id, List<Id>> accountToOpportunity = new Map<Id, List<Id>>();
+
+        if (currentOpportunities.isEmpty()) {
+            return accountToOpportunity;
+        }
+
+        for (Opportunity eachOppty : currentOpportunities) {
+            if (eachOppty.AccountId != null
+                && !oppAccountDetails.get(eachOppty.AccountId).npe01__SYSTEMIsIndividual__c) {
```

Comment:
why not use your AutomatedSoftCreditService.isOrganizationalAccount() method (which I still which took an account rather than the flag :-) )

Reply:
Added the usage of the isOrganizationalAccount method.